BrainPortal/lib/cbrain_task_generators/schema_task_generator.rb

```diff
@@ -166,6 +165,12 @@ def integrate(register: true, create_tool_config: false, multi_version: false)
         })[partial]
       end
 
+      # Add a helper method for accessing the help file (for use on the tools page)
+      # Written by the rake task cbrain:plugins:install, within the subtask public_assets
+      helpFileName = name + "_help.html" 
+      helpFileDir  = File.join( "cbrain_plugins", "cbrain_tasks", "help_files/" )
```

Comment:
I am adding a method to the task class that points to where the help files should be written out by the rake task. Then I call that method in the Tools table (to make the help button). I thought it was a bit better than hard-coding the path, in case we want to call the method else where (e.g. tests). 

Because Boutiques tools are parts of plugins, I put the help files there, under the 'public' dir (which, annoyingly to me, is implict in rails paths like this).